peps/pep-0763.rst

```diff
@@ -0,0 +1,275 @@
+PEP: 763
+Title: Limiting deletions on PyPI
+Author: William Woodruff <william@yossarian.net>,
+        Alexis Challande <alexis.challande@trailofbits.com>
+Sponsor: Donald Stufft <donald@stufft.io>
+PEP-Delegate: Donald Stufft <donald@stufft.io>
+Discussions-To: TODO
+Status: Draft
+Type: Standards Track
+Topic: Packaging
+Created: 07-Oct-2024
+Post-History: `09-Jul-2022 <https://discuss.python.org/t/stop-allowing-deleting-things-from-pypi/17227>`__,
+              `01-Oct-2024 <https://discuss.python.org/t/pre-pep-limiting-deletions-on-pypi/66351>`__
+
+Abstract
+========
+
+This PEP proposes limiting user-level deletions of projects (including release
+and file-level deletions) on PyPI, in favor of the "yanking" mechanism
+adopted by PyPI with :pep:`592`.
+
+In particular, the PEP proposes a time-based deletion criterion via
```

Comment:
This sentence reads a bit awkwardly to me. Perhaps clarify or break into 2 sentences.